src/frontend/komponenter/Fagsak/Forhåndsvarsel/Brukeruttalelse.tsx

```diff
@@ -0,0 +1,151 @@
+import type { ForhåndsvarselFormData } from './useForhåndsvarselMutations';
+import type { UseFormReturn } from 'react-hook-form/dist/types/form';
+
+import {
+    VStack,
+    RadioGroup,
+    HStack,
+    Radio,
+    DatePicker,
+    TextField,
+    Textarea,
+    useDatepicker,
+} from '@navikt/ds-react';
+import { ATextWidthMax } from '@navikt/ds-tokens/dist/tokens';
+import { format } from 'date-fns/format';
+import React from 'react';
+import { Controller, useWatch } from 'react-hook-form';
+
+import { HarBrukerUttaltSeg } from './Enums';
+type Props = {
+    methods: UseFormReturn<ForhåndsvarselFormData>;
+    kanUtsetteFrist?: boolean;
+};
+
+export const Brukeruttalelse: React.FC<Props> = ({ methods, kanUtsetteFrist = false }) => {
+    const {
+        register,
+        control,
+        formState: { errors },
+    } = methods;
+
+    const harBrukerUttaltSeg = useWatch({
+        control: methods.control,
+        name: 'harBrukerUttaltSeg',
+    });
+
+    const uttalelsesdatoDatepicker = useDatepicker({
```

Comment:
Blir det overflødig med datoDate?
```suggestion
    const uttalelseDatepicker = useDatepicker({
```
```suggestion
    const uttalelsesdatepicker = useDatepicker({
```